setup.py

```diff
@@ -31,6 +31,7 @@
             "transformers==4.18.0",
             "protobuf==3.19.4",
             "Pillow==8.4.0",
+            "pydicom==2.3.1",
```

Comment:
This shouldn't be an entry in `test`, it should have its own keyword in `extras_require`

Reply:
done